artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/connect/mirror/AckManager.java

```diff
@@ -139,7 +140,14 @@ public void endRetry() {
 
       // schedule a retry
       if (!sortRetries().isEmpty()) {
-         scheduledComponent.delay();
+         ActiveMQScheduledComponent scheduleComponentReference = scheduledComponent;
+         if (scheduleComponentReference != null) {
+            try {
+               scheduleComponentReference.delay();
+            } catch (RejectedExecutionException thatsOK) {
+               logger.debug(thatsOK.getMessage(), thatsOK);
```

Comment:
Logging _thatsOk_ already prints its message, so rather than simply duplicating it, adding a useful description message would be helpful to anyone that later encounters this logging and stacktrace. E.g to understand _what_ was rejected without having to go look up the code.